bofhound/ad/models/bloodhound_domain.py

```diff
@@ -69,15 +73,21 @@ def __init__(self, object):
 
 
     def to_json(self, only_common_properties=True):
+        self.Properties['isaclprotected'] = self.IsACLProtected
         domain = super().to_json(only_common_properties)
 
         domain["ObjectIdentifier"] = self.ObjectIdentifier
         domain["Trusts"] = self.Trusts
+        domain["ContainedBy"] = None
         # The below is all unsupported as of now.
         domain["Aces"] = self.Aces
         domain["Links"] = self.Links
         domain["ChildObjects"] = self.ChildObjects
+        
+        self.GPOChanges["AffectedComputers"] = self.AffectedComputers
+        self.GPOChanges["AffectedUsers"] = self.AffectedUsers
```

Comment:
```json
"data": [{
        "GPOChanges": {
            "LocalAdmins": [],
            "RemoteDesktopUsers": [],
            "DcomUsers": [],
            "PSRemoteUsers": [],
            "AffectedComputers": [{
                "ObjectIdentifier": "S-1-5-21-1800740480-423033261-719922928-1001",
                "ObjectType": "Computer"
            }, {
                "ObjectIdentifier": "S-1-5-21-1800740480-423033261-719922928-1123",
                "ObjectType": "Computer"
            }, {
                "ObjectIdentifier": "S-1-5-21-1800740480-423033261-719922928-1124",
                "ObjectType": "Computer"
            }, {
                "ObjectIdentifier": "S-1-5-21-1800740480-423033261-719922928-1125",
                "ObjectType": "Computer"
            }, {
                "ObjectIdentifier": "S-1-5-21-1800740480-423033261-719922928-1126",
                "ObjectType": "Computer"
            }, {
                "ObjectIdentifier": "S-1-5-21-1800740480-423033261-719922928-1127",
                "ObjectType": "Computer"
            }, {
                "ObjectIdentifier": "S-1-5-21-1800740480-423033261-719922928-1128",
                "ObjectType": "Computer"
            }, {
                "ObjectIdentifier": "S-1-5-21-1800740480-423033261-719922928-1129",
                "ObjectType": "Computer"
            }, {
                "ObjectIdentifier": "S-1-5-21-1800740480-423033261-719922928-1130",
                "ObjectType": "Computer"
            }, {
                "ObjectIdentifier": "S-1-5-21-1800740480-423033261-719922928-1131",
                "ObjectType": "Computer"
            }, {
                "ObjectIdentifier": "S-1-5-21-1800740480-423033261-719922928-1132",
                "ObjectType": "Computer"
            }]
        },
        "Properties": {
            "domain": "GALAXY.LAN",
            "name": "GALAXY.LAN",
            "distinguishedname": "DC=GALAXY,DC=LAN",
            "domainsid": "S-1-5-21-1800740480-423033261-719922928",
            "isaclprotected": false,
            "description": null,
            "whencreated": 1708427711,
            "functionallevel": "2016",
            "collected": true
        },
        "ChildObjects": [],
        "Trusts": [{
            "TargetDomainSid": "S-1-5-21-1431803989-2292373293-1963840564",
            "TargetDomainName": "REBELS4.GALAXY.LAN",
            "IsTransitive": true,
            "SidFilteringEnabled": false,
            "TrustDirection": "Bidirectional",
            "TrustType": "ParentChild"
        }, {
            "TargetDomainSid": "S-1-5-21-1093034243-2474785251-74389104",
            "TargetDomainName": "LOGRES.LAN",
            "IsTransitive": true,
            "SidFilteringEnabled": false,
            "TrustDirection": "Outbound",
            "TrustType": "Forest"
        }],
        "Links": [{
            "IsEnforced": false,
            "GUID": "300E9FA7-92C9-4681-A8CA-3407BB03881A"
        }],
        "Aces": [{
```

This comes from Sharphound `XXXXXXXX_domains.json` output from my personnal lab

I don't know how important is this but I think keeping this could be a good idea ?